lib/HTTP/BrowserDetect.pm

```diff
@@ -141,65 +206,91 @@ our @MISC_TESTS = qw(
+    curl                                 => 'curl',
+    'developers.google.com//web/snippet' => 'Google+ Snippet',
+    embedly                              => 'Embedly',
+    facebook                             => 'Facebook',
+    facebookexternalhit                  => 'Facebook',
+    flipboard                            => 'Flipboard',
+    getright                             => 'GetRight',
+    golib                                => 'Go language http library',
+    google                               => 'Googlebot',
+    googleadsbot                         => 'Google AdsBot',
+    googleadsense                        => 'Google AdSense',
+    googlebotimage                       => 'Googlebot Images',
+    googlebotnews                        => 'Googlebot News',
+    googlebotvideo                       => 'Googlebot Video',
+    googlefavicon                        => 'Google Favicon',
+    googlemobile                         => 'Googlebot Mobile',
+    indy                                 => 'Indy Library',
+    infoseek                             => 'InfoSeek',
+    ipsagent                             => 'Verisign ips-agent',
+    java                                 => 'Java',
+    lib                                  => 'lib',
+    linkchecker                          => 'LinkChecker',
+    linkedinbot                          => 'LinkedIn',
+    linkexchange                         => 'LinkExchange',
+    lwp                                  => 'LWP::UserAgent',
+    lycos                                => 'Lycos',
+    malware                              => 'Malware / hack attempt',
+    mj12bot                              => 'Majestic-12 DSearch',
+    msn                                  => 'MSN',
+    msnmobile                            => 'MSN Mobile',
+    msoffice                             => 'Microsoft Office',
+    nutch                                => 'Apache Nutch',
+    nuzzel                               => 'Nuzzel',
+    outbrain                             => 'Outbrain',
+    phplib                               => 'PHP http library',
+    'pinterest/0.'                       => 'Pinterest',
+    'pinterestbot'                       => 'Pinterest',
+    puf                                  => 'puf',
+    'quora link preview'                 => 'Quora Link Preview',
+    redditbot                            => 'reddit',
+    robot                                => 'robot',
+    rogerbot                             => 'Moz',
+    rubylib                              => 'Ruby http library',
+    showyoubot                           => 'Showyou',
+    slackbot                             => 'slack',
+    slurp                                => 'Yahoo! Slurp',
+    specialarchiver                      => 'archive.org_bot',
+    tumblr                               => 'Tumblr',
+    twitterbot                           => 'Twitter',
+    url                                  => 'url',
```

Comment:
Also seems prone to false positives.

Reply:
Yeah, I just removed this.  I didn't see the value in it.  It also removes the associated method.  We'll see if anyone complains.